include/envoy/http/header_map.h

```diff
@@ -144,13 +139,20 @@ class HeaderString {
   /**
    * @return whether a substring exists in the string.
    */
-  bool find(const char* str) const { return strstr(c_str(), str); }
+  bool find(const char* str) const {
```

Comment:
take string_view as parameter? Though I'm not sure how much value for this method existing here as getStringView().find() is essentially same.

Reply:
Definitely, I have a list of follow ups going, this was one of them. I am thinking it makes the most sense to convert callers to `getStringView().find()` and then we can eliminate `find()` here.